vignettes/bambu.Rmd

```diff
@@ -204,25 +239,31 @@ individual genes. Here we look at gene ENSG00000099968 and visualise the
 transcript coordinates for annotated and novel isoforms and expression levels 
 for these isoforms across all samples. 
 ```{r, fig.width = 8, fig.height = 10}
-plotBambu(se, type = "annotation", gene_id = "ENSG00000099968")
+plotBambu(se.multiSample, type = "annotation", gene_id = "ENSG00000099968")
 ```
 
 ### Obtain gene expression estimates from transcript expression
 {#gene-expression}
-Gene expression can be calculated from the transcript expression estimates
-returned by *bambu* using the \code{transcriptToGeneExpression} function.
-Looking at the output, we can see there are novel genes identified as well
+
+For the downstream analysis, we will add the condition of interest to the 
+\code{colData} object that describes the samples. Here we are interested in
+a comparison of the 2 cell lines:
+```{r}
+colData(se.multiSample)$condition <- as.factor(SGNexSamples$cellLine)
+```
+
+To obtain the accurate gene expression estimates which uses all reads that can be assigned to each gene (including reads that are incompatible with all existing annotations) you can run the following command. Looking at the output, we can see there are novel genes identified as well
 ```{r}
-seGene <- transcriptToGeneExpression(se)
-seGene
+seGene.multiSample <- transcriptToGeneExpression(se.multiSample)
```

Comment:
Error in `:=`(TXNAME, "Incomaptible") : 
Check that is.data.table(DT) == TRUE. Otherwise, := and `:=`(...) are defined for use in j, once only and in particular ways. See help(":=").

Reply:
for this one, when you use se.multiSample, does it still give you error?